lib/src/value/string.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {hash} from 'immutable';
+
+import {Value} from './value';
+import {valueError} from '../utils';
+
+/** A SassScript string. */
+export class SassString extends Value {
+  private readonly textInternal: string;
+  private readonly hasQuotesInternal: boolean;
+
+  /** Creates a string with `text`, optionally with quotes. */
+  constructor(text: string, options?: {/** @default true */ quotes: boolean}) {
```

Comment:
I know you know this, but reminder to make this optional once https://github.com/sass/sass/pull/3151 lands.

Reply:
Thanks for the reminder :)